cluster/cluster.yaml

```diff
@@ -272,6 +272,13 @@ Resources:
           FromPort: 30443
           IpProtocol: tcp
           ToPort: 30444
+{{- end }}
+{{- if index .Cluster.ConfigItems "open_cdp_proxy_port" }}
```

Comment:
At this point I'd switch to something like `open_worker_ports: "30443-30444,30445-30445"` and then use templating to generate it.